src/services/translations/translation.service.ts

```diff
@@ -10,7 +10,7 @@ export class WizdomTranslationService implements IWizdomTranslationService {
         if(this.translations){
             var translation = this.translations[key];            
             if(translation == null) { // Missing translation                
-                if(this.wizdomdevelopermode && key) {
+                if(this.wizdomdevelopermode && key && !key.startsWith("channel_")) {
```

Comment:
I understand the intend with this change. 
But wizdom-services is build to be generic across all our modules. This change will only help for one specific use case inside a single module.
Instead we could implement it with optional arguments for the translate method or an alternate translate method. 

Something like: 
translate(key: string, ignoreMissingTranslation: bool = false)
or 
translateAndIgnoreMissingTranslation(key: string)
or
translateAndSuppressErrors(key: string)

Other ideas and namings are welcome.